packages/cli/commands/sandbox/delete.js

```diff
@@ -91,6 +94,14 @@ exports.handler = async options => {
     })
   );
 
+  if (isDefaultAccount) {
+    logger.log(
```

Comment:
Adds an extra warning to let user know they are deleting a default account, and updates the success message to say default account was deleted. Confirmed with Erin that this provides better context as to why theyre being prompted with another default account selection:
<img width="833" alt="Screen Shot 2022-10-11 at 17 41 30" src="https://user-images.githubusercontent.com/16788677/195195195-6efe2d18-3f49-4340-8182-b1514bb3b983.png">